tests/unit/tracking.tcl

```diff
@@ -247,16 +249,17 @@ start_server {tags {"tracking network logreqres:skip"}} {
         for {set i 0} {$i <= $MAX_TRIES && $res < 0} {incr i} {
             set res [lsearch -exact [r PING] "tracking-redir-broken"]
         }
-        assert {$res >= 0}
-        # Consume PING reply
-        assert_equal PONG [r read]
```

Comment:
why this fail? Because the r is not in resp3?

Reply:
I suspect that it did somehow. maybe the 'r' is disconnected and reconnected at some point

Reply:
let's give it a try.